missas/core/templates/home.html

```diff
@@ -0,0 +1,215 @@
+                <div class="card-body text-center">
+                    <i class="fa-solid fa-map-marker-alt"
+                       style="font-size: 2rem;
+                              color: #2E7D32;
+                              margin-bottom: 0.5rem"></i>
+                    <h4 class="card-title mb-1" style="color: #2E7D32; font-weight: 700;">{{ stats.cities_with_parishes }}</h4>
+                    <p class="card-text small text-muted mb-0">Cidades com paróquias</p>
+                </div>
+            </div>
+        </div>
+        <div class="col-6 col-md-3 mb-3">
+            <div class="card h-100 border-0 shadow-sm"
+                 style="background: linear-gradient(135deg, #FFF3E0, #FFE0B2)">
+                <div class="card-body text-center">
+                    <i class="fa-solid fa-church"
+                       style="font-size: 2rem;
+                              color: #F57C00;
+                              margin-bottom: 0.5rem"></i>
+                    <h4 class="card-title mb-1" style="color: #F57C00; font-weight: 700;">{{ stats.parishes }}</h4>
+                    <p class="card-text small text-muted mb-0">Paróquias cadastradas</p>
+                </div>
+            </div>
+        </div>
+        <div class="col-6 col-md-3 mb-3">
+            <div class="card h-100 border-0 shadow-sm"
+                 style="background: linear-gradient(135deg, #F3E5F5, #E1BEE7)">
+                <div class="card-body text-center">
+                    <i class="fa-solid fa-clock"
+                       style="font-size: 2rem;
+                              color: #7B1FA2;
+                              margin-bottom: 0.5rem"></i>
+                    <h4 class="card-title mb-1" style="color: #7B1FA2; font-weight: 700;">{{ stats.schedules }}</h4>
+                    <p class="card-text small text-muted mb-0">Horários disponíveis</p>
+                </div>
+            </div>
+        </div>
+        <div class="col-6 col-md-3 mb-3">
+            <div class="card h-100 border-0 shadow-sm"
+                 style="background: linear-gradient(135deg, #E8F5E8, #A5D6A7)">
+                <div class="card-body text-center">
+                    <i class="fa-solid fa-check-circle"
+                       style="font-size: 2rem;
+                              color: #388E3C;
+                              margin-bottom: 0.5rem"></i>
+                    <h4 class="card-title mb-1" style="color: #388E3C; font-weight: 700;">{{ stats.verified_schedules }}</h4>
+                    <p class="card-text small text-muted mb-0">Horários verificados</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <!-- Estados e Cidades -->
+    <div class="row mb-4">
+        <div class="col-12">
+            <h3 class="mb-3" style="color: var(--primary-blue); font-weight: 600;">
+                <i class="fa-solid fa-map me-2"></i>
+                Estados e Cidades com Paróquias
```

Comment:
Invert the order of the stats and states with parishes sections. States with parishes should come first.